force-app/main/default/objects/Contact/Contact.object-meta.xml

```diff
@@ -323,9 +323,9 @@
         <type>Default</type>
     </actionOverrides>
     <compactLayoutAssignment>SA_Contact_Compact_Layout</compactLayoutAssignment>
-    <enableFeeds>true</enableFeeds>
+        <enableFeeds>true</enableFeeds>
     <enableHistory>true</enableHistory>
-    <externalSharingModel>ControlledByParent</externalSharingModel>
+    <externalSharingModel>Private</externalSharingModel>
```

Comment:
@deepakmulamalla Was this requirement from Business to make Contact object Private and not controlled by Account which they have access to? How are opening the contact access if you set it as Private?

Reply:
@cgijeffolsen This was done as per of issue fix where Contacts were not visible for portal users.
@ckleinacn